docs/openapi_spec.yaml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.0
+openapi: 3.1.0
```

Comment:
Need version 3.1 to have support for adding descriptions of properties of a schema whose type is a `$ref` reference of another schema (without having to use the old `anyOf` keyword workaround), see [discussion](https://github.com/OAI/OpenAPI-Specification/issues/1514#issuecomment-1385551628).